### ISSUE TITLE
can I install core modules with cpanfile?

### ISSUE THREAD

**Issue by ufobat** [skaji/cpm#140]
I have got a centos perl 5.16 that does not include all core modules :-(

A module that happens to be installed on my build box is `JSON::PP`. `JSON::PP` is also in my `cpanfile`. It seems that cpm is not installing `JSON::PP` into `local` when I invoke this command:

```
# perl cpm/cpm install -L local --target-perl 5.16.3 --with-develop # tried that before
perl cpm/cpm install -L local --with-develop
```

My problem is that I want to fatpack my application an the JSON::PP of the centos does not contain a packlist, therefore I wanted  cpm to install JSON::PP as well. I could install JSON::PP with another cpm call, but that is somehow a hack because I want all the modules to be fatpacked in oder to run on other/different "strange" system perls.

Is there a way to tell cpm to install things in `local` even if they're available already?

**Comment by skaji** [comment]
> I think I am looking for something similar to --self-contained from cpanm. 

I don't think so.

**Comment by skaji** [comment]
While I understand your motivation, I think cpm may assume perl installation to be with the complete set of core modules.

As you said, you can always install JSON::PP separately.
Or, if you intend to execute cpm with `--target-perl 5.16.3` (on which JSON::PP is shipped with version 2.27200),
you may write `requires "JSON::PP", "2.97001";` in cpanfile.

**Comment by ufobat** [comment]
thank you :+1:
